packages/proto-loader/src/index.ts

```diff
@@ -126,6 +126,8 @@ export interface MethodDefinition<RequestType, ResponseType, OutputRequestType=R
   originalName?: string;
   requestType: MessageTypeDefinition;
   responseType: MessageTypeDefinition;
+  options?: Protobuf.IMethod['options'];
+  parsedOptions?: Protobuf.IMethod['parsedOptions'];
```

Comment:
I don't like using the type exported by Protobuf.js here. I would prefer to have the type declared directly in this file. It looks like both of these fields have the type `{ [k: string]: any }`. That way, if Protobuf.js changes this field type some time in the future, we don't accidentally end up exporting a type we didn't intend to export here.